cypress/integration/pages/mediaAssetPage/tests.js

```diff
@@ -30,6 +31,29 @@ export const testsThatFollowSmokeTestConfig = ({ service, pageType }) => {
         },
       );
     });
+
+    it('should render a timestamp', () => {
+      cy.request(`${config[service].pageTypes[pageType].path}.json`).then(
+        ({ body }) => {
+          const { lastPublished, firstPublished } = body.metadata;
+          cy.get('time')
+            .eq(0)
+            .should('exist')
+            .should('be.visible')
+            .should('have.attr', 'datetime')
+            .should('not.be.empty');
+
+          if (lastPublished !== firstPublished) {
+            cy.get('time')
+              .eq(1)
+              .should(
+                'contain',
+                appConfig[service].default.articleTimestampPrefix,
+              );
+          }
```

Comment:
This logic to append the `Updated` prefixed to a timestamp needs to be confirmed with product for the MAPs

Reply:
We can tweak it later if it changes, just wanna integrate some timestamps for now, this is easiest